src/test/java/com/github/onsdigital/babbage/api/endpoint/util/LegacyProxyMaxAgeTest.java

```diff
@@ -0,0 +1,78 @@
+package com.github.onsdigital.babbage.api.endpoint.util;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.mockStatic;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.client.methods.RequestBuilder;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.message.BasicHeader;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.MockedStatic;
+
+import com.github.onsdigital.babbage.api.util.LegacyProxyMaxAge;
+
+public class LegacyProxyMaxAgeTest {
+    private CloseableHttpClient mockHttpClient;
+    private CloseableHttpResponse mockResponse;
+    private HttpServletRequest mockRequest;
+    private RequestBuilder mockRequestBuilder;
```

Comment:
This and the following property are only used inside `setup`, so they should be moved there.